app/sections/videos/video-details.component.ts

```diff
@@ -0,0 +1,51 @@
+import {Component} from 'angular2/core';
+import {OnActivate, RouteData, RouteParams, ComponentInstruction} from 'angular2/router';
+import {ContentfulService} from 'ng2-contentful/dist/src/services/contentful.service';
+import {ContentfulConfig} from '../../app.constans';
+import {ContentfulTypes as ct} from 'ng2-contentful/dist/ng2-contentful';
+import {transformResponse} from '../../shared/tools/response.tools';
+import {EntriesView} from '../../shared/components/entries-view/entries-view.component';
+
+@Component({
+  template: <string> require('./video-details.component.html'),
+  styles: [`
+    .entry-content {
+      margin-bottom: 25px;
+    }
+  `],
+  host: {
+    class: 'content'
+  },
+  directives: [EntriesView]
+})
+export class VideoDetails implements OnActivate {
+  private content: any;
+
+  constructor(private _params: RouteParams,
+              private _contentful: ContentfulService) {
+  }
+
+  routerOnActivate(to: ComponentInstruction, from: ComponentInstruction): any {
```

Comment:
looks similar to what we have in `about.component.ts`  - Can we create separate service for this and reuse this code in all appropriate components?